backend/drm/drm.c

```diff
@@ -1255,6 +1255,15 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 			}
 		}
 
+		memset(&wlr_conn->props, 0, sizeof(wlr_conn->props));
```

Comment:
We can do all of this only if `drm_conn->connection == DRM_MODE_CONNECTED`. There's no point in doing it for disconnected connectors.